### PR TITLE
[CLOUD-3074] Expose Subsystem metrics

### PR DESCRIPTION
**Pull request by jmesnil**
* add the metric subsystem to standalone-openshift.xml
* by default, prefix the subsystem metrics with jboss
* add wildfly.statistics-enabled=true System property to
  openshift-launch.sh script

JIRA: https://issues.jboss.org/browse/CLOUD-3074

### CODE REVIEW

**Comment by jmesnil** [comment]
@luck3y This PR is built on top of #59 that updates the standalone-openshift.xml for EAP CD 16.

Once #59 is merged, this PR will only contain the 
https://github.com/jboss-container-images/jboss-eap-modules/pull/60/commits/2c25f2a21300cbb44ba0ee92344f391c645aadc9 commit that adds the `microprofile-metrics-smallrye` extension and subsystem to `standalone-openshift.xml`

**Comment by rsvoboda** [comment]
subsystem metrics prefix change is CD16 image only change.

https://github.com/jbossas/jboss-eap7/blob/CD16/microprofile/metrics-smallrye/src/main/resources/subsystem-templates/microprofile-metrics-smallrye.xml uses `wildfly` prefix

File: jboss-eap-cd16-openshift/added/standalone-openshift.xml [hunk 1]
```diff
@@ -480,6 +481,10 @@
             <!-- ##MICROPROFILE_CONFIG_SOURCE## -->
         </subsystem>
         <subsystem xmlns="urn:wildfly:microprofile-health-smallrye:1.0"/>
+        <subsystem xmlns="urn:wildfly:microprofile-metrics-smallrye:2.0"
+            security-enabled="false"
+            exposed-subsystems="*"
+            prefix="${wildfly.metrics.prefix:jboss}"/>
```
Comment (psakar) [inline jboss-eap-cd16-openshift/added/standalone-openshift.xml]: Should it be jboss-eap instead of jboss ?

Comment (jmesnil) [reply jboss-eap-cd16-openshift/added/standalone-openshift.xml]: Metrics that are exposed by EAP (using Prometheus JMX agent) already use the `jboss` prefix. https://github.com/jboss-container-images/jboss-eap-modules/blob/master/jboss/container/eap/prometheus/config/7.2/artifacts/opt/jboss/container/prometheus/etc/jmx-exporter-config.yaml#L43

Comment (psakar) [reply jboss-eap-cd16-openshift/added/standalone-openshift.xml]: That something is already done does not mean it is correct

Comment (psakar) [reply jboss-eap-cd16-openshift/added/standalone-openshift.xml]: there is no product jboss only jboss-eap

**Pull request merged**